db/models/Events.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose')
+
+const categories = ['music', 'dining']
+const accessibilityOptions = ['braille-menu', 'wheelchair-friendly']
+
+const eventSchema = mongoose.Schema(
+  {
+    ownerId: { type: mongoose.Schema.Types.ObjectId, ref: 'User' },
+    name: { type: String, required: true },
+    category: { type: [String], enum: categories, required: true },
```

Comment:
just checked- as it is doesn't work, but ```category: [{ type: String, enum: categories}]``` does work. In order to require at least one value, a custom validator would have to be implemented though, as even with ```required: true```, it would allow an empty list @m4v15